src/index.html

```diff
@@ -11,8 +11,46 @@
       rel="stylesheet"
       href="./styles/index.scss"
     />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    />
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div
+      class="card"
+      data-qa="card"
+    >
+      <div class="card__imac">
+        <img
+          src="images/imac.jpeg"
+          alt="imac"
```

Comment:
The 'alt' attribute should be more descriptive of the image content. It should provide a meaningful description in case the image is not displayed.